ui-common/src/components/modals/TrafficStatsModal/TimelineBarChart/TimelineBarChart.tsx

```diff
@@ -42,58 +42,61 @@ export const TimelineBarChart: React.FC<TimelineBarChartProps> = ({ timeLineBarC
 
     useEffect(() => {
         if (selectedProtocol === ALL_PROTOCOLS) {
-            setCommandStats(null);
-            setcommandNames(null);
+            setMethodsStats(null);
+            setMethodsNamesAndColors(null);
             return;
         }
         const commandsNames = [];
```

Comment:
if you rename command with method, I think it would be better to change in all the places